core/modules/utils/dom/scroller.js

```diff
@@ -49,10 +49,14 @@ Handle an event
 */
 PageScroller.prototype.handleEvent = function(event) {
 	if(event.type === "tm-scroll") {
+		var options = {};
+		if(event.paramObject && event.paramObject.hasOwnProperty("animationDuration")) {
```

Comment:
We've got a wrapper for hasOwnProperty `$tw.utils.hop()`

Reply:
Good catch, thank you. Now resolved.